### DISCUSSION TITLE
How to set app Badge with Capacitor ?

### DISCUSSION THREAD

**Discussion by Aarbel** [ionic-team/capacitor#4154] [Q&A]
Is their a way to change app badge number with basic Capacitor plugins ? Or do we need to install cordova plugin for that ?

![image](https://user-images.githubusercontent.com/25119847/106876760-5cc02f00-66d8-11eb-89c2-05b76f41e027.png)

Cf https://ionicframework.com/docs/native/badge and https://github.com/katzer/cordova-plugin-badge

Thanks a lot for your help !

**Comment by robingenz** [comment]
I've created a plugin for Capacitor: https://github.com/robingenz/capacitor-badge
Give it a try. It is in an early stage of development. Many features are planned. I appreciate any feedback.

**Reply by Aarbel** [reply]
Looks great, did you submitted it to the capacitor team ? Which parts are missing ?

**Reply by robingenz** [reply]
Not yet, I will move the repo to the capacitor community soon. Version 1.0.0 will be released tomorrow. But there are no more changes planned for this version.
The plugin supports Android, iOS and PWA.

**Reply by robingenz** [reply]
I hope the plugin is what you were looking for and I could answer this discussion for you with it.

**Reply by Aarbel** [reply]
I remember Android had limitations with the cordova plugins (limitations to show badges and to show app icon when receiving notifications).

For these subjects, the most important is the documentation to use well the apis, with some UI examples (which are often missing with cordova / ionic plugins)

Providing badges for capacitor electron would also be a must !
https://github.com/capacitor-community/electron

**Reply by robingenz** [reply]
There are limitations. These are mentioned under [quirks](https://github.com/robingenz/capacitor-badge#quirks). To display the badges I use [ShortcutBadger](https://github.com/leolin310148/ShortcutBadger) on Android.

I hope the documentation is clear and easy to understand.

Electron support is not currently planned. However, contributions are welcome.

**Reply by Aarbel** [reply]
Seems great with the quirks doc !

**Reply by Aarbel** [reply]
This plugin should really be part of Capacitor main repo.

About electron Capacitor team should talk with https://todesktop.com/, they provide a standard api / builder for desktop apps https://docs.todesktop.com/
